test/summary.t

```diff
@@ -223,6 +223,28 @@ W\d{1,2} \d{4}-\d{2}-\d{2} .{3}       ?0:00:00 0:00:00 0:00:00 0:00:00
 [ ]+0:00:00
 """)
 
+    def test_with_multiple_multi_day_tracks(self):
+        """Summary should display intervals that span multiple days"""
+        self.t("track 1980-01-01T01:00:00 - 1980-01-03T01:00:00 proja")
```

Comment:
I would prefer to use relative dates instead of fixed ones, so one has an extra bit of randomness in the test data. 
Fixed dates are easier to debug but you do not have the chance to spot those errors that only occur when your test date is X days after special date Y...